lib/gs1/barcode/healthcare.rb

```diff
@@ -40,6 +42,14 @@ def validate(level)
         validate_highest
       end
 
+      def validate_records
+        record_diff = @params_order - self.class.record_names
```

Comment:
Could there be optional fields in the barcode that we might want to ignore, or are the defined fields the only ones allowed according to the specification?

Reply:
Seems to be some additional fields according to GS1 standards:
![image](https://user-images.githubusercontent.com/1053201/63768221-ec509400-c8cf-11e9-8eed-81d922f6a1ff.png)

In that case we need to implement additional records and add them to `define_records` for `GS1::Barcode::Healthcare`

Reply:
Is it possible to get a list of all allowed records?

Reply:
I believe this is a different issue and should be solved in another pull request.

Reply:
I've done some testing (with this branch as well as master) and it seems like we already have this issue since a lot of records are not yet implemented, which makes parsing fail silently if a barcode includes an AI that is not implemented. Better left for another PR as you say.